src/abstractdataframe/iteration.jl

```diff
@@ -13,17 +13,13 @@ with each row represented as a `DataFrameRow`.
 
 A value of this type is returned by the [`eachrow`](@ref) function.
 """
-struct DataFrameRows{D<:AbstractDataFrame, S} <: AbstractVector{DataFrameRow{D, S}}
+struct DataFrameRows{D<:AbstractDataFrame} <: AbstractVector{DataFrameRow}
```

Comment:
The old eltype was incorrect so I changed it to abstract one.

Reply:
What do you mean by "incorrect"?

Reply:
It was too specific and in some corner cases it was - well - incorrect 😄.